packages/background/package.json

```diff
@@ -28,6 +28,8 @@
     "@ethersproject/hash": "^5.7.0",
     "@ethersproject/transactions": "^5.7.0",
     "@ethersproject/wallet": "^5.7.0",
+    "@ethereumjs/tx": "^3.5.2",
+    "@ethereumjs/common": "^2.6.5",
```

Comment:
You should use `ethers.js`. Let me know if there is issue

Reply:
`
import {
  AccessListEIP2930Transaction,
  FeeMarketEIP1559Transaction,
  Transaction,
  TypedTransaction,
} from "@ethereumjs/tx";
`
`ethers.js` does not provide these.